.travis.yml

```diff
@@ -81,13 +106,15 @@ jobs:
         - OSX_VERSION=10.14
       install:
         - .travis/install_conda.sh "Miniconda3-latest-MacOSX-x86_64.sh"
-      script: .travis/conda_build_and_deploy.sh --label 'dev'
-    - name: "conda linux python3.6"
+      script: .travis/conda_build_and_deploy.sh
+    - name: "release conda linux python3.6"
```

Comment:
Why not just add the one line script from here to the "release conda linux" job?
That would remove a lot of duplicated lines.

It also makes no sense to have release builds on a cron job.